accepted/00078-maintenance-windows-web-calendar.md

```diff
@@ -0,0 +1,87 @@
+- Feature Name: Maintenance Windows Web Calendar
+- Start Date: 2020-12-18
+- RFC PR: (leave empty) 
+
+# Summary
+[summary]: #summary
+
+Provide the user with a calendar view that displays information about the maintenance windows associated with the calendar.
+
+# Motivation
+[motivation]: #motivation
+
+### Why are we doing this?
+Currently we display the contents of the ical file uploaded by the users in the calendar details and creation menus. Since it's difficult to draw information about the maintenance windows defined in that file from the pure ical we want to provide this information displayed as a calendar.
+
+### What use cases does it support?
+
+When choosing a maintenance schedule from a maintenance calendar the user needs to know the details of the maintenance windows contained in the calendar.
+
+Typical workflow when creating new maintenance windows:
+
+1. Head to `Schedule` -> `Maintenance Windows` -> `Calendars` -> `Create`
+2. Follow the steps to create a new calendar
+3. Go to `Schedules` to create a new maintenance schedule
+4. Use the information provided in the created calendars details view to pick a maintenance window
+5. Assign systems to the newly created maintenance schedule
+
+### What is the expected outcome?
+
+Replace the plain ical text in the calendar details and creation menu with a visual representation of the maintenance windows contained in the uploaded ical file.
+Use the created calendar view to also display information about a maintenance window in the maintenance schedule details and creation menu. 
```

Comment:
Is it correct to say that this RFC is just about the visualization of maintenance windows, and that the creation will still happen via upload of an iCal file?

Reply:
For now it will. Current scope is to only change the way Maintenance schedules/calendar are displayed to the user. The creation is something that can be added in the future. I put into consideration if the investigated component option would be suitable for that as well.